doc/configuration.rst

```diff
@@ -82,7 +81,7 @@ Global options:
     }
 
 .. note::
-    New in version 3.12
+    New in version 3.13
```

Comment:
https://github.com/ultrabug/py3status/blob/3.12/doc/configuration.rst

tags are great :)

Also with the doc breakage, you should really be building the docs when you make changes.  Also when we merger PRs too because now I have all this again

```
reading sources... [100%] py3                                                                    
modules-info.inc:83: WARNING: Unexpected indentation.
modules-info.inc:84: WARNING: Block quote ends without a blank line; unexpected unindent.
modules-info.inc:88: WARNING: Definition list ends without a blank line; unexpected unindent.
modules-info.inc:5623: WARNING: Block quote ends without a blank line; unexpected unindent.
looking for now-outdated files... none found
pickling environment... done
checking consistency... /home/toby/dev/py3status/doc/formatter.rst: WARNING: document isn't included in any toctree
done
preparing documents... done
```
Anyhow I think I'm the only one who actually cares :(

Reply:
If you're satisfied with this version, can you merge this? Thx.

Reply:
no because you have changed the version and if you look at 3.12 release (that url) you can see it was there

or do `git checkout 3.12` you may need to do a `git fetch` first

Reply:
Whatever you're smoking... I want some.

Reply:
commit 15509e61f2a86ea646d01f4fe5c0e0d7b4e18477 so actually from v3.5

stop whining and do what you're told

Reply:
I fix my mistake in #1411... so actually from ~~3.12~~ 3.13.

Reply:
ok in that case it is in the wrong place move it under storage